Sources/Templates/Extensions/Method+Extension.swift

```diff
@@ -246,6 +263,28 @@ private extension Method {
         }
         return parts.compactMap { $0 }.joined(separator: " ")
     }
+
+    func functionName() -> String {
+        // Generics should now be added manually
```

Comment:
This comment doesn't really make sense to me, perhaps elaborate a bit

Reply:
So previously you could call `name` (line 240) and get the full function signature, the functionname, generic and parameters between brackets, but because the opaque types are not supported in the parameters i have to manually construct the function signature. for now i did it for the function name and parameters, but the generic `<T: Encodable>` part should also be but between those things now manually. That's still a TODO for this branch